tests/views/test_message.py

```diff
@@ -1163,8 +1163,7 @@ def test_reopening_conversation_returns_to_closed_tab(self, mock_request, mock_g
         response_body = response.data.decode("utf-8").replace(" ", "")
 
         # validate that the currently selected tab is as expected (i.e aria-current="location")
-        match_str = '"aria-current="location">Closed'
-        self.assertIn(match_str, response_body)
+        self.assertRegex(response_body, r'"aria-current="location"\s*>Closed<')
```

Comment:
Newlines and whitspace were causing test issues and we weren't correctly testing this as the conversation tab was not being passed through. Updated to properly assert on the active tab